packages/core/src/controllers/SwapController.ts

```diff
@@ -245,9 +259,14 @@ export const SwapController = {
   },
 
   switchTokens() {
+    if (state.initializing || !state.initialized) {
```

Comment:
do we need this check in other methods ?

Reply:
Nope, this this for disabling functionality when users click the switch button on the center because that button is still visible even thought the swap is initializing (when have skeletons view)